Artesian/Artesian.SDK/Service/Query/IQueryWithFill.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) ARK LTD. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for
+// license information. 
+using NodaTime;
+
+namespace Artesian.SDK.Service
+{
+    interface IQueryWithFill<T>: IBaseQuery<T>
```

Comment:
Name is misleading as it also include RelativePeriod. This is sloppy ...

Either rename it, or split in 2 interfaces or any other clean solution :)